### PR TITLE
staticfile.org CDN

### PR DESCRIPTION
**Pull request by TraderStf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -218,6 +218,7 @@ Table of Contents
   * https://developers.google.com/speed/libraries/ - The Google Hosted Libraries is a content distribution network for the most popular, open-source JavaScript libraries.
   * https://www.asp.net/ajax/cdn - The Microsoft Ajax Content Delivery Network (CDN) hosts popular third party JavaScript libraries such as jQuery and enables you to easily add them to your Web application
   * https://toranproxy.com/ - Proxy for Packagist and GitHub. Never fail CD. Free for personal use, 1 developer, no support.
+  * http://staticfile.org/index_en.html - Free for Open Source: libraries, js, css, images, swf... Can add yours, by (paying) Qiniu in China.
```
Comment (ripienaar) [inline README.md]: seems this CDN is china based only? sure it works from elsewhere but extremely slow, not sure its generally helpful for this list?

**Comment by TraderStf** [comment]
For non-EU people :-)

**Comment by ripienaar** [comment]
not non-eu, for one specific country, dont think it fits the bill sorry

**Comment by TraderStf** [comment]
Can not compute :-)

They have 7 servers worldwide and the OS CDN is in English.

全球化架构
七牛首创全球分布式架构，充分考虑了跨地域数据存储和访问场景，可以接受数据中心之间低速且不稳定的连接网络。

**Pull request closed without merging**